backend/src/db/alert_trigger_snapshot_integration_test.go

```
@@ -0,0 +1,60 @@
+func TestAlertTriggerSnapshotMigrationDatabaseSemantics(t *testing.T) {
+	dsn := strings.TrimSpace(os.Getenv("AGRAFA_RLS_TEST_DSN"))
+	if dsn == "" {
+		t.Skip("AGRAFA_RLS_TEST_DSN is not set")
+	}
+
+	ctx := context.Background()
+	db, err := appdb.OpenPostgres(ctx, dsn)
+	if err != nil {
+		t.Fatalf("open postgres: %v", err)
+	}
+	defer db.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `db.Close` is not checked

(errcheck)